src/libs/ajax.js

```diff
@@ -55,7 +55,7 @@ const withCancellation = wrappedFetch => async (...args) => {
   }
 }
 
-const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
+const withRetryOnError = _.curry((errorsToIgnore, wrappedFetch) => async (...args) => {
```

Comment:
A more flexible way to do this might be to have `withRetryOnError` accept a function instead of a list of properties.

```js
const withRetryOnError = _.curry((shouldRetry, wrappedFetch) ...
...
if (!shouldRetry(error)) {
  throw error
}
```